src/utils.js

```diff
@@ -263,7 +263,7 @@ let toolpath = global.packageInfo.package
 let processes = [];
 function execTool(tool, args, callback) {
   let pid = exec(
-    [path.join(toolpath, tool)].concat(args).join(" "),
+    [path.join(toolpath, tool).replace(" ", " ")].concat(args).join(" "),
```

Comment:
Erm does this replace a space with a space?

Reply:
lol, missed a backslash there. how the fuck did it work on my machine?  :D i created a new user with a space in the name and everything

Reply:
You going to push an update? :)